test/mlir/accelerators/nnpa/driver/compiler-config.mlir

```diff
@@ -0,0 +1,16 @@
+
+// RUN: onnx-mlir --maccel=NNPA -v -tag="test" %s -o %t 2>&1 | FileCheck %s
+
+// -----
+
+// REQUIRES: system-s390
```

Comment:
Where did you know to use "system-s390"? I could not find it in llvm repository. 
When I check "system-s390" on z14, the lit test is excluded. You can check on your side, by looking the result of `make check-onnx-lit`:
```
Testing Time: 4.54s
  Unsupported:   5
  Passed     : 235
```
It a lit test is excluded, it will go to "Unsupported"

Reply:
You can use `// REQUIRES: system-linux` instead to run the test on linux. Or use `lit.local.cfg` to test on big endian machine only: https://github.com/onnx/onnx-mlir/blob/main/test/mlir/accelerators/nnpa/module_op_be/lit.local.cfg

Reply:
I thought system take the name of the machine from `uname` command. This test can only work on machine with zDNN library, not just zDNN.h. I will try system-linux plus big endian. Now I recalled that I wanted to add my test into module_op_be directory. That's the right place to share the local config.

Reply:
I guess "system-s390" does not exist, so there is no system satisfying the requirement, so the test will be disabled in all environment.